src/Umbraco.Core/PropertyEditors/ImageCropperConfiguration.cs

```diff
@@ -22,4 +26,35 @@ public class Crop
             public int Height { get; set; }
         }
     }
+
+    internal static class ImageCropperConfigurationExtensions
+    {
+        /// <summary>
+        /// Applies the configuration to ensure only valid crops are kept and have the correct width/height.
+        /// </summary>
+        /// <param name="configuration">The configuration.</param>
+        public static void ApplyConfiguration(this ImageCropperValue imageCropperValue, ImageCropperConfiguration configuration)
```

Comment:
I've moved this internal method from the `ImageCropperValue` to an extension method, so it aligns with the other `ApplyConfiguration` method and lives nicely next the the configuration it applies.